src/main/java/net/neoforged/neoforge/codec/NeoforgeNbtCodecs.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.codec;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.DataResult;
+import com.mojang.serialization.Dynamic;
+import net.minecraft.nbt.NbtOps;
+import net.minecraft.nbt.Tag;
+
+public class NeoforgeNbtCodecs {
```

Comment:
I'd just put this in NeoForgeExtraCodecs.

Reply:
This kind of thing is definitely opinionated; my stance on it is that having a class like `ExtraCodecs` means it becomes a catch-all and grows into a monster after a while... which means either you keep adding on or finally split it and make a giant breaking change.

Similar to the reason I'm adding this stuff to the `codec` package instead of putting stuff in `common.util` -- discoverability.

Reply:
The advantage of a catch all is that people know where to look. Introducing a new class for a single method in the hope of adding more later sounds like typical YAGNI.

Reply:
This doesn't need its own new class, we want the helpers to be discoverable.